.cirrus/tasks.yml

```diff
@@ -169,7 +169,7 @@ chart_testing_task:
     - kubectl create secret docker-registry pullsecret --namespace test --docker-username=${DOCKER_USERNAME} --docker-password=${DOCKER_PASSWORD} --dry-run=client -o yaml | kubectl apply -f -
   script:
     - ct lint --config test.yaml --all
-    - ct install --namespace test --config test.yaml --all
+    - ct install --namespace test --config test.yaml --all --debug
```

Comment:
Do we want to keep the `--debug`?

Reply:
Yes, we discussed this. It is intentional: without this flag and the extra-argument added to underlying helm installation (see `test.yaml`) we don't have enough information to understand the error in case of failures. Furthermore, enabling it does not add much more logging in case of success.